src/test/java/sorinfoca/Tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,54 @@
+package sorinfoca.Tests.Tema6;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import sorinfoca.driver.BrowserManager;
+
+import java.util.Set;
+
+public class DemoQaWindow {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQaWindowPage();
+        clickNewTabButton();
+        closeBrowser();
+
+    }
+
+    public static void navigateToDemoQaWindowPage() {
+        driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/browser-windows");
+        System.out.println("Am deschis Demo QA window page");
+    }
+
+    public static void clickNewTabButton() {
```

Comment:
Try to click on other options too and make separated methods for them too.Thanks!